htdocs/install/mysql/tables/llx_expeditiondet.sql

```diff
@@ -24,7 +24,7 @@ create table llx_expeditiondet
   rowid             integer AUTO_INCREMENT PRIMARY KEY,
   fk_expedition     integer NOT NULL,  						-- ID of parent object
   fk_element        integer,           						-- ID of main source object
-  fk_origin_line    integer,           						-- ID of line of source object (proposal, sale order). TODO should be renamed into fk_elementdet in SQL files and code in same PR
+  fk_elementdet    integer,           						-- ID of line of source object (proposal, sale order)
```

Comment:
If you update the field in table used to create a new instance, you must also add a line into file htdocs/install/mysql/migration/19.0.0-20.0.0.sql to migrate the field name.

Reply:
Are there interractions with the sql database in other files?
Normally, the principle of the model is that you only use it for sql interractions, right? For other files, we use the object properties. That's why, for the moment, I've left the fk_origin_line property, but put the value of the new field in it.

Reply:
@eldy I must not be very clever, but I can't find anything about the old sql field in the files. Can you give me an example of a file where the old sql field is still present?

Reply:
Ηι @zephyriony 

You have to also modify at least following files so not to break things

https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/install/mysql/tables/llx_expeditiondet.key.sql
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/install/mysql/migration/19.0.0-20.0.0.sql
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/expedition/card.php
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/commande/class/api_orders.class.php
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/commande/class/commande.class.php
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/product/class/product.class.php
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/delivery/class/delivery.class.php
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/product/stock/replenish.php
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/core/modules/modExpedition.class.php
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/product/reassort.php
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/expedition/list.php
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/expedition/dispatch.php
https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/core/lib/sendings.lib.php

Reply:
Hi @zephyriony 
Can you fix conflicts so  i can help and push changes to your branch to complete this?

Reply:
Hi @sonikf 
Yes thank you it would be great I have less time to deal with it now but I think it is an important point for the future version of Dolibarr ^^
I solved the conflict normally :)